src/Options.php

```diff
@@ -925,9 +975,13 @@ private function configureOptions(OptionsResolver $resolver): void
             'in_app_include' => [],
             'send_default_pii' => false,
             'max_value_length' => 1024,
+            'transport' => null,
+            'http_client' => null,
             'http_proxy' => null,
+            'http_proxy_authentication' => null,
```

Comment:
Didn't we support proxy authentication before? We might otherwise need to make sure you couldn't pass the `http_proxy` as `user:pass@proxyhost.example` or we have a BC, not a big one, but keeping it simpler might be worth splitting `http_proxy` on the `@` in the client instead of adding another option for it.

Reply:
Imo, we did not. Referring to https://curl.se/libcurl/c/CURLOPT_PROXY.html

Reply:
Huh, interesting, okay!